File: examples/resource-raw/resource-raw.py
import json
import iam_floyd as statement

s = (
    # doc-start
    statement.S3() \
    .allow() \
    .all_actions() \
    .on('arn:aws:s3:::some-bucket',
        'arn:aws:s3:::another-bucket')
    # doc-end
).to_json()

pretty = json.dumps(s, indent=4, sort_keys=True)
print(pretty)

File: examples/resource-raw/resource-raw.ts
import { deploy, out } from '../../helper/typescript/typescript_test';
import { Statement } from '../../lib';

function getStatement() {
  return (
    // doc-start
    new Statement.S3() //
      .allow()
      .allActions()
      .on(
        'arn:aws:s3:::example-bucket', //
        'arn:aws:s3:::another-bucket',
      )
    // doc-end
  );
}
const s = [getStatement()];
out(s);
deploy(s);
